test/unit/Fixture/AnonymousClassInNamespace.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
A possible scenario missing is `NestedAnonymousClasses`:

```php
<?php
return new class () {
    function getWrapped($index) {
        if (0 === $index) {
            return new class() {};
        }

        return new class() {};
    }
}
```

This would verify that:

 * we analyze nested class declaration nodes (in this case, within an `if` block)
 * at runtime, we know the line where a reflection class was generated (depending on `$index` here)